src/main/java/uk/gov/hmcts/reform/civil/notification/BundleCreationApplicantNotificationHandler.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.reform.civil.notification;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.civil.config.properties.notification.NotificationsProperties;
+import uk.gov.hmcts.reform.civil.handler.callback.camunda.notification.NotificationData;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.service.NotificationException;
+import uk.gov.hmcts.reform.civil.service.NotificationService;
+
+import java.util.Map;
+
+@Service
+@RequiredArgsConstructor
+public class BundleCreationApplicantNotificationHandler implements NotificationData {
+
+    private static final String REFERENCE_TEMPLATE = "evidence-upload-notification-%s";
+    private final NotificationService notificationService;
+    private final NotificationsProperties notificationsProperties;
+
+    public void notifyApplicantBundleCreation(CaseData caseData) throws NotificationException {
+        //Send email to Applicant
+        notificationService.sendMail(
+            caseData.getApplicantSolicitor1UserDetails().getEmail(),
+            notificationsProperties.getBundleCreationTemplate(),
+            addProperties(caseData),
+            String.format(REFERENCE_TEMPLATE, caseData.getLegacyCaseReference())
+        );
+    }
+
+    @Override
+    public Map<String, String> addProperties(CaseData caseData) {
+        return Map.of(
+            CLAIM_REFERENCE_NUMBER, caseData.getLegacyCaseReference()
```

Comment:
The ticket seems to show an email template with 3 values:
1. claimant v defendant
2. claim reference number
3. Legal Rep Name

Yet I only see the claim reference number here - is the template in the ticket not up to date?